data/classificator.txt

```diff
@@ -90,17 +90,23 @@ world  +
         parcel_locker  -
         parking  +
             fee  +
-                lane  -
                 street_side  -
+                underground  -
                 {}
-            lane  -
```

Comment:
parking=lane is deleted still in the auto-generated files;
did you forget to regenerate?